### PR TITLE
Reload using Queries.find_by

### PR DESCRIPTION
**Pull request by cjcolvar**

### CODE REVIEW

**Comment by cjcolvar** [comment]
Wait on merging this...I have a rubocop issue.

File: app/actors/hyrax/actors/file_set_actor.rb [hunk 1]
```diff
@@ -77,11 +77,11 @@ def create_metadata(file_set_params = {})
       def attach_to_work(work, file_set_params = {})
         acquire_lock_for(work.id) do
           # Ensure we have an up-to-date copy of the members association, so that we append to the end of the list.
-          work.reload if work.persisted?
+          Hyrax::Queries.find_by(id: work.id) if work.persisted?
```
Comment (jcoyne) [inline app/actors/hyrax/actors/file_set_actor.rb]: The return value of this query isn't used.

**Comment by cjcolvar** [comment]
Wait until #2081 is merged then I'll resolve the conflicts.

**Comment by cjcolvar** [comment]
@jcoyne I resolved the conflict using github's UI.  Do you want me to squash instead?

**Comment by cjcolvar** [comment]
Hold on...I need to fix something else.

**Comment by cjcolvar** [comment]
@jcoyne Ok, this time I think it's good.

**Pull request merged**